src/ALCProxy/CommObjects.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+
+namespace ALCProxy
+{
+    interface IServerObject
+    {
+        //public void ReceiveMethod(MethodInfo method, object[] args);
+        public void ReturnResult(Type t, object instance);
```

Comment:
I would avoid adding the `public` to the interface method.

Reply:
Even more, it's not possible to specify an access modifier for interface members.

Reply:
@YohDeadfall Yes it is. This is apart of the new C# 8.0 [default interfaces](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-8.0/default-interface-methods.md) feature. Specifying access is now supported on interfaces.

Reply:
Thanks for pointing it out! But there is an open question about `protected` and `internal` modifiers because it doesn't look natural on interfaces which are designed to be fully public or fully internal.